internal/controllers/suite_test.go

```diff
@@ -50,6 +51,12 @@ var (
 	ctx             = ctrl.SetupSignalHandler()
 )
 
+func init() {
+	utilruntime.Must(feature.MutableGates.SetFromMap(map[string]bool{
+		string(feature.PropagateLabels): true,
```

Comment:
Just for my understanding: does this indicate that the PropagateLabels feature flag is mutable i.e. it can be turned on or off by the end user?

Reply:
No, unless they build it with this code. Once the FG enabled by default, it can’t be disabled unless there is a handler like this exposed to the user. The `Gates` is immutable, so this has to happen before it is injected in the variable inside `init()`. This line here just satisfies the existing tests expectation on the FG remaining being set to true.

Reply:
Sorry @yiannistri let me correct myself - it is in fact mutable. We expose flag `--feature-gates` to override the defaults: 
```
      --feature-gates mapStringBool            A set of key=value pairs that describe feature gates for alpha/experimental features. Options are:
                                               AllAlpha=true|false (ALPHA - default=false)
                                               AllBeta=true|false (BETA - default=false)
                                               ClusterResourceSet=true|false (BETA - default=true)
                                               ClusterTopology=true|false (ALPHA - default=false)
                                               KubeadmBootstrapFormatIgnition=true|false (ALPHA - default=false)
                                               MachinePool=true|false (BETA - default=true)
                                               MachineSetPreflightChecks=true|false (ALPHA - default=false)
                                               RuntimeSDK=true|false (ALPHA - default=false)
                                               etcd-snapshot-restore=true|false (BETA - default=false)
                                               managementv3-cluster=true|false (BETA - default=false)
                                               propagate-labels=true|false (BETA - default=false)
                                               rancher-kube-secret-patch=true|false (BETA - default=false)
```
Although I’m seeing some leftover features in the list :)